packages/inter-protocol/src/collectFees.js

```diff
@@ -1,4 +1,4 @@
-import { atomicTransfer } from '@agoric/zoe/src/contractSupport';
+import { atomicTransfer } from '@agoric/zoe/src/contractSupport/index.js';
```

Comment:
TIL that ESM doesn't support the implicit `index.js` so we should have this everywhere. 👍